meetings/2019-06-19.md

```diff
@@ -0,0 +1,69 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-06-19
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=ve9x5aanbfU   
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/717
+
+## Present
+
+* Anatoli Papirovski @apapirovski (TSC)
+* Beth Griggs @BethGriggs (IBM, guest participant)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Gabriel Schulhof @gabrielschulhof (TSC)
+* Gireesh Punathil @gireeshpunathil (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Michaël Zasso @targos (TSC)
+* Sam Roberts @sam-github (TSC)
+
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* https://github.com/nodejs/node/issues/27731 - Nominating Jiawen Geng @gengjiawen to be a Collaborator . Gireesh and Matteo will be onboarding Jiawen on 06/20
+
+### nodejs/node
+
+* benchmark: refactor buffer benchmarks [#26418](https://github.com/nodejs/node/pull/26418)
+  * Gabriel, is an option to run a different set each time ?
+  * Michael, still too long even for a particular category.
+  * Agreed from those in the meeting to move to vote unless there is objection to that in the issue
+    itself
+
+* http2: make compat finished match http/1 [#24347](https://github.com/nodejs/node/pull/24347)
+  * Anatoli was in the meeting and will review and comment in the issue today. 
+
+### nodejs/TSC
+
+* Nominating @BethGriggs to the TSC [718](https://github.com/nodejs/TSC/pull/718)
+  * FYI - https://github.com/nodejs/TSC/issues/718
+  * Michael will add Beth to observers
+
+* doc: update TSC charter [#698](https://github.com/nodejs/TSC/pull/698)
+  * waiting for the CPC to approve
```

Comment:
I thought they did already. Am I mistaken? /ping @MylesBorins

Reply:
Board reviewed. CPC has not yet had a meeting with quorum of voting members. Working with CPC to document process and ensure everything is consistent for future changes